### DISCUSSION TITLE
Jsx divs are not render

### DISCUSSION THREAD

**Discussion by RogeMateos** [PrismJS/prism#3829] [General]
Hello everyone,

I'm encountering an issue while trying to render some JSX code with <div> elements, but it doesn't seem to be working as expected. I'm using a CDN and working on CodeSandbox. I've attached the link to the project and included some screenshots for reference.

Any help would be greatly appreciated!
[Project Link](https://yldfz5.csb.app/)

Many thanks!

![image](https://github.com/user-attachments/assets/3962921a-94e4-42b8-a27a-5922231cbd71)

**Comment by LadyBeGood** [comment]
This is a common problem. JSX is similar to HTML in syntax, so browser tries to execute it instead of rendering it like normal text. 

Straightforward but tedious solution is to escape the special characters like `<` and `>` with `&lt;` and `&gt;` respectively. So your code will look something like:
```jsx
return (
    &lt;div className="box"&gt;
        &lt;h1&gt;ComponentD&lt;/h1&gt;
        &lt;h2&gt;{`Hello ${user}`}&lt;/h2&gt;
    &lt;/div&gt;
);
```


Yet another simple solution is to use this plugin with was made to solve this problem: [unescaped-markup](https://prismjs.com/plugins/unescaped-markup/). I haven't tried this plugin myself but it appears to be the answer to your problem.
